### PR TITLE
The memory service doesn't check swap, add a swap service

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

File: pack/commands.cfg [hunk 1]
```diff
@@ -32,6 +32,11 @@ define command {
 }
 
 define command {
+    command_name  check_linux_swap
+    command_line  $PLUGINSDIR$/check_snmp_mem.pl -s -w $_HOSTMEMORY_WARN$ -c $_HOSTMEMORY_CRIT$ -- -v 2c -c $_HOSTSNMPCOMMUNITY$ $HOSTADDRESS$
```
Comment (Seb-Solon) [inline pack/commands.cfg]: should be _HOSTSWAP_WARN and _HOSTSWAP_CRIT I think :)

**Comment by Seb-Solon** [comment]
Seems fine except my comment . Thanks for help

**Comment by dmsimard** [comment]
Good call.. copy paste error.

**Comment by Seb-Solon** [comment]
Thanks!

**Pull request merged**